### ISSUE TITLE
[Feature Request] Support feedbackd

### ISSUE THREAD

**Issue by Spirk-Stan** [dino/dino#1142]
Dino has support for notification sounds for messages if compiled with the notification-sound plugin enabled. It's limited, though. It isn't influenced by the Do Not Disturb state of the environment, it doesn't utilize the force feedback motors of devices that support it (like the Pinephone), and there's no audible notifications for incoming calls.

Using a feedback service like feedbackd would resolve all those issues. It wouldn't have to be exclusive to it, it's just the one I've been using with my Pinephone.

Documentation: https://developer.puri.sm/Librem5/Software_Reference/Environments/Phosh/Audio_Haptic.html

Source code: https://source.puri.sm/Librem5/feedbackd

**Comment by carlosgonz0** [comment]
Yes will nice if Dino supporting "feedbackd" for notification under Phosh.
